.github/workflows/build-store-ios.yml

```diff
@@ -41,6 +41,9 @@ jobs:
       - name: Disable widget
         if: ${{ env.ENABLE_WIDGET != 'true' }}
         run: bundle exec configure_extensions remove ios/atb.xcodeproj app departureWidget AtbAppIntent
+      - name: Enable apple pass suppression
+        if: matrix.org == 'fram'
```

Comment:
Here as a small improvement we could do the same as `if: ${{ env.ENABLE_WIDGET != 'true' }}` defining a .env var with this is more dynamic in a way if in ATB we want to disable this in the future.

Reply:
I agree! I've added `ENABLE_APPLE_PASS_SUPPRESSION` as an environment variable. I sat it to false for all environments except FRAM production.